### DISCUSSION TITLE
priority setting for threads created by uv_thread_create

### DISCUSSION THREAD

**Discussion by hhu8** [libuv/libuv#4050] [Ideas]
Hello,

I am Node.js application developer and I found that we could take advantage of the Node.js worker thread feature. For some tasks like logging, event tracking and some CPU intensive tasks (with lower business value), we want to make them running in separate threads and with lower priority, so that they won't contend with the main thread, which is dealing with the user requests. For some more critical tasks like heat beating, we want to make them running with higher priority.
Node.js is calling the uv_thread_create() method in libuv to create worker thread, and there is no parameter or API to set priority.
Could we add this parameter or API in libuv?

Best regards,
Hao

**Comment by bnoordhuis** [comment]
For reference, tracked in #4051.

**Comment by hhu8** [comment]
tracked in https://github.com/libuv/libuv/issues/4051.
